src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -0,0 +1,445 @@
+            width: 0;
+            height: 0;
+            border: solid 5px transparent;
+            background: transparent;
+            border-top: solid 7px #fff;
+            border-bottom-width: 0;
+            margin-top:1px;
+            cursor: pointer;
+            float: right;
+            margin-right: 50px;
+            margin-top: 5px;
+        }
+    </style>
+    <nav class="row">
+        <button
+            type="button"
+            class="txlist-view-btn btn radio left checked">
+            History
+        </button>
+        <button
+            type="button"
+            class="utxo-view-btn btn radio right">
+            UTXO
+        </button>
+    </nav>
+    <div class="pagination-container">
+        <a class="pagination-back pagination-arrow pagination-disabled">&#8249;</a>
+        Page <input class="pagination-idx" type="number" step="1" min="1" value="1"></input><span class="pagination-counter"></span>
+        <a class="pagination-next pagination-arrow pagination-disabled">&#8250;</a>
+        <br>
+    </div>
+    <span class="page-limit-label">Page limit: </span><fieldset class="page-limit-fieldset"> 
+        <select class="page-limit-select">
+            <option value="10">10</option>
+            <option value="25">25</option>
+            <option value="50" selected>50</option>
+            <option value="100">100</option>
+            <option value="0">All</option>
+        </select>
+    </fieldset>
+    <br>
+    <div class="search-container">
+        <input class="search" type="text" placeholder="Search..." />
+    </div>
+    <br>
+    <table class="tx-table">
+        <thead>
+            <tr>
+                <th value="category" class="category-header"><div class="category-arrow"></div></th>
+                <th value="txid" class="optional txid-header">TxID<div class="txid-arrow"></div></th>
+                <th value="label" class="optional label-header">Label<div class="label-arrow"></div></th>
+                <th value="amount" class="amount-header">Amount<div class="amount-arrow"></div></th>
+                <th value="time" class="time-header">Time<div class="time-arrow up-arrow"></div></th>
+                <th value="confirmations" class="confirmations-header">Confirmations<div class="confirmations-arrow"></div></th>
+                <th value="blockhash" class="optional hidden blockhash-header">
+                    <div class="tool-tip">{% include "includes/merkletooltip.html" %}</div>
+                    Block Hash
+                </th>
+            </tr>
+    </thead>
+    <tr class="empty">
+        <td></td>
+        <td>Fetching transactions...</td>
+        <td class="optional"></td>
+        <td class="optional"></td>
+        <td></td>
+        <td></td>
+        <td class="optional hidden blockhash-empty"></td>
+    </tr>
+	<tbody class="tx-tbody">
+	</tbody>
+    </table>
+</template>
+
+<script type="text/javascript">
+    class TxTableElement extends HTMLElement {
```

Comment:
Would be good to have some comments on the code. Class-documentation and maybe each function or so?

Reply:
Awesome! So much better, really!